_posts/0000-01-02-Miha07.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Hey!
```

Comment:
Stephen Strange: Let me ask you this one time… What master do you serve?
Peter Quill: What master do I serve? What am I supposed to say, “Jesus”?
Tony Stark: You’re from Earth.
Peter Quill: I’m not from Earth, I’m from Missouri.
Tony Stark: Yeah, that’s on Earth, dipsh*t.